pkgs/tools/filesystems/dwarfs/default.nix

```diff
@@ -0,0 +1,110 @@
+{ stdenv
+, bison
+, boost
+, cmake
+, double-conversion
+, fetchFromGitHub
+, fmt_8
+, fuse3
+, gflags
+, git
+, glog
+, gtest
+, jemalloc
+, lib
+, libarchive
+, libevent
+, libunwind
+, lz4
+, openssl
+, pkg-config
+, ronn
+, xxHash
+, xz
+, zstd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dwarfs";
+  version = "0.6.1";
+  src = fetchFromGitHub {
```

Comment:
There's usually a newline after `pname` and `version` (but it doesn't really matter).

Reply:
I'll implement what Sandro recommended (i.e. LF between `version` and `src`) instead of this.

Reply:
Oh, that's what I meant, no clue why I wrote so confusingly. Sorry!